products/sle/main.pm

```diff
@@ -55,6 +55,10 @@ sub is_update_test_repo_test {
     return get_var('TEST') !~ /^mru-/ && (get_var('FLAVOR', '') =~ /-Updates$/);
 }
 
+sub is_bridged_networking {
+    return !(check_var('VIRSH_VMM_FAMILY', 'kvm') or (check_var('BACKEND', 'svirt') and check_var('ARCH', 's390x')));
```

Comment:
@Soulofdestiny I thought of using `S390_ZKVM` variable instead for the second part, but I guess it's not set at this point, right?